### PR TITLE
Add Prettier config

### PR DESCRIPTION
**Pull request by joseluispiedrahitamrf**

### CODE REVIEW

File: packages/prettier-config/index.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 120,
+  "tabWidth": 4,
```
Comment (diegofernandezmrf) [inline packages/prettier-config/index.json]: By default prettier use spaces for indentation. Does this conflict with this eslint rule https://github.com/Marfeel/coding-standards/blob/master/packages/eslint-config-js/index.js#L130?

File: packages/prettier-config/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (diegofernandezmrf) [inline packages/prettier-config/package.json]: Should we have prettier as a peerDependency? I know this package would do nothing if prettier is not installed, but at least we can specify a prettier version (e.g.: `<2.0.0` or `^2.0.0`)

File: packages/prettier-config/index.json (hunk 1 above)
Comment (joseluispiedrahitamrf) [reply packages/prettier-config/index.json]: This actually refers to: ["the number of spaces per indentation-level"](https://prettier.io/docs/en/options.html#tab-width), not tabs vs spaces. That said, I'd prefer a `tabWidth` of 2 (the default value).

Comment (diegofernandezmrf) [reply packages/prettier-config/index.json]: I always use my editor on half screen so I prefer 2 as well. But that is just personal preference.

**Pull request merged**